Orange/widgets/data/owpythonscript.py

```diff
@@ -645,7 +667,7 @@ def saveScript(self):
             f.close()
 
     def initial_locals_state(self):
-        d = dict([(i.name, getattr(self, i.name, None)) for i in self.inputs])
+        d = dict([(name, getattr(self, name, None)) for name in self.inputs_names])
```

Comment:
I think it'd be better to pass lists, not dicts. The script doesn't know the meaning of id's, so it would always use just the values.